lua/everblush/config.lua

```diff
@@ -102,64 +102,65 @@ M.highlights_base = function (colors)
-    TSKeywordReturn = { fg = colors.color4 },
-    TSLabel = { fg = colors.color4 },
-    TSMethod = { fg = colors.color12 },
-    TSNamespace = { fg = colors.color9 },
-    TSNumber = { fg = colors.color3 },
-    TSParameter = { fg = colors.color1 },
-    TSParameterReference = { fg = colors.color9 },
-    TSProperty = { fg = colors.color1 },
-    TSPunctDelimiter = { fg = colors.color7 },
-    TSPunctBracket = { fg = colors.color7 },
-    TSPunctSpecial = { fg = colors.color7 },
-    TSRepeat = { fg = colors.color11 },
-    TSString = { fg = colors.color2 },
-    TSStringRegex = { fg = colors.color2 },
-    TSStringEscape = { fg = colors.color4 },
-    TSStringSpecial = { fg = colors.color4 },
-    TSSymbol = { fg = colors.color1 },
-    TSTag = { fg = colors.color4 },
-    TSTagAttribute = { fg = colors.color1 },
-    TSTagDelimiter = { fg = colors.color7 },
-    TSText = { fg = colors.color7 },
-    TSStrong = { fg = colors.color7 },
-    TSEmphasis = { italic = true, fg = colors.color7 },
-    TSUnderline = { fg = colors.color5 },
-    TSStrike = { fg = colors.color7 },
-    TSTitle = { fg = colors.color3 },
-    TSLiteral = { fg = colors.color2 },
-    TSURI = { fg = colors.color3 },
-    TSMath = { fg = colors.color6 },
-    TSTextReference = { fg = colors.color6 },
-    TSEnvirontment = { fg = colors.color5 },
-    TSEnvironmentName = { fg = colors.color3 },
-    TSNote = { fg = colors.color8 },
-    TSWarning = { fg = colors.color0, bg = colors.color1 },
-    TSDanger = { fg = colors.color8 },
-    TSType = { fg = colors.color3 },
-    TSTypeBuiltin = { fg = colors.color3 },
-    TSVariable = { fg = colors.color7 },
-    TSVariableBuiltin = { fg = colors.color4 },
+    ["@attribute"] = { fg = colors.color4 },
+    ["@boolean"] = { fg = colors.color6 },
+    ["@character"] = { fg = colors.color4 },
+    ["@comment"] = { fg = colors.comment, italic = true },
+    ["@conditional"] = { fg = colors.color1 },
+    ["@constant"] = { fg = colors.color6 },
+    ["@constant.builtin"] = { fg = colors.color4 },
+    ["@constant.macro"] = { fg = colors.color3 },
+    ["@constructor"] = { fg = colors.color4 },
+    ["@exception"] = { fg = colors.color8 },
+    ["@field"] = { fg = colors.color1 },
+    ["@float"] = { fg = colors.color8 },
+    ["@function"] = { fg = colors.color1 },
+    ["@func.builtin"] = { fg = colors.color14 },
+    ["@func.macro"] = { fg = colors.color2 },
+    ["@include"] = { fg = colors.color9 },
+	["@operator"] = { fg = colors.color7 },
+    ["@keyword"] = { fg = colors.color5 },
+    ["@keyword.function"] = { fg = colors.color4 },
+    ["@keyword.operator"] = { fg = colors.color12 },
+    ["@keyword.return"] = { fg = colors.color4 },
+    ["@label"] = { fg = colors.color4 },
+    ["@method"] = { fg = colors.color12 },
+    ["@namespace"] = { fg = colors.color9 },
+    ["@number"] = { fg = colors.color3 },
+    ["@parameter"] = { fg = colors.color1 },
+    ["@parameter.reference"] = { fg = colors.color9 },
```

Comment:
"@parameter.reference" doesn't seem to exist according to this: https://github.com/nvim-treesitter/nvim-treesitter/blob/master/CONTRIBUTING.md

"@reference" and "@text.reference" do exist though, but I am not sure of which one is the correct one.

Reply:
Did some more digging. First of all, "@reference" is not a group used for highlighting. "@text.reference" is, but it is used for footnotes, citations, etc. so I doubt.

I cannot find anything recent about this "@parameter.reference" (or TSParameterReference) that is recent. Maybe this highlighting group does not exist anymore and can be deleted? Sorry, I'm not familiar with nvim-treesitter and I cannot find more documentation, so I cannot provide more info

Reply:
That's already plenty of information, thanks. I'll probably end up deleting it after asking in nvim-dev.

Reply:
Hmm I didn't get a clear answer, but I think we should keep it

Reply:
It's staying.